src/util/Util.js

```diff
@@ -279,11 +279,11 @@ class Util {
    */
   static resolveColor(color) {
```

Comment:
This should really just be:
```suggestion
  static resolveColor(color) {
    const uColor = color.toUpperCase();
```
and then reuse that variable.

Reply:
^ however inside the assertion that color is a string, not an array or number